<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1301</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Submeter</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1271</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">image: url(:/imagens/noname.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Determine o bairro onde está localizado o problema:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="BoxBairro">
                 <property name="minimumSize">
                  <size>
                   <width>490</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>490</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <item>
                  <property name="text">
                   <string>  ---</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Agua Fria</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Altiplano</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Alto do Mateus</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Bairro das Indústrias</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Bairro dos Estados</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Bancários</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Bessa</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Brisamar</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cabo Branco</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Castelo Branco</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Centro</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Colinas do Sul</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cristo Redentor</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Cruz das Armas</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Ernesto Geisel</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Funcionários</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Grotão</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Jaguaribe</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>José Américo</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Manaíra</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mangabeira</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Miramar</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Penha</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Portal do Sol</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Quadramares</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Rangel</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Róger</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>São José</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Tambaú</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Tambauzinho</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Torre</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Valentina Figueiredo</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                     <weight>50</weight>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Data do ocorrido: </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="inputDataOcorrido">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Agora, qual o problema que você deseja relatar:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="boxProblema">
               <property name="minimumSize">
                <size>
                 <width>490</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>490</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>  ---</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Alto índice de poluição nas ruas</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Obras abandonadas</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Pouca sinalização (semáforos, placas, etc)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Problemas elétricos (fios caídos, falta de energia recorrente, etc)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Problemas sanitários (vazamentos de esgoto, falta d'água, etc)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Vias obstruídas (buracos, árvores obstruindo, etc)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Outro</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>229</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonCadastrar">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Submeter queixa</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Todas as queixas:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tabelaGeral">
             <property name="minimumSize">
              <size>
               <width>600</width>
               <height>0</height>
              </size>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Bairro</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Data da queixa</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Data do ocorrido</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Problema</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btnOrdenarPorBairro">
             <property name="text">
              <string>Organizar por bairro</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAntigas">
             <property name="text">
              <string>Mais antigas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRecentes">
             <property name="text">
              <string>Mais recentes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);</string>
           </property>
           <property name="text">
            <string>Clique duas vezes na linha para editar.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>661</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Bairro com o maior número de incidentes urbanos:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tabelaEstatisticaBairro">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Bairro</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Quantidade</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Número de queixas relatadas no total:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="numQueixas">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1322</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSalvar">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuSalvar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
